api/auth.go

```diff
@@ -73,7 +72,7 @@ func (a *API) parseJWTClaims(bearer string, r *http.Request, w http.ResponseWrit
 	})
 	if err != nil {
 		a.clearCookieTokens(config, w)
-		return nil, unauthorizedError("Invalid token: %v", err)
+		return nil, unauthorizedError("invalid token: %v", err)
```

Comment:
After giving this some thought I think we should just return something like `invalid JWT: unable to parse or verify signature` while logging `err` instead of returning it. wdyt?